package.json

```diff
@@ -119,7 +120,7 @@
     "fbjs": "^0.8.16",
     "hoist-non-react-statics": "^2.3.1",
     "invariant": "^2.2.2",
-    "lodash": "4.17.4",
+    "lodash": "^4.17.4",
```

Comment:
Enabling treeshaking resulted in all of react-apollo's lodash version being bundled into main.js :/  Not sure why.  Relaxing this constraint allows web to use a single lodash version.

Reply:
Mainline react-apollo no longer uses the single lodash library, so this won't be a problem once we upgrade: https://github.com/apollographql/react-apollo/blob/master/package.json#L150-L156